cellprofiler/modules/filterobjects.py

```diff
@@ -367,6 +380,14 @@ def help_settings(self):
     def visible_settings(self):
         result =[self.target_name, self.object_name, 
                  self.spacer_2, self.mode]
+        if self.mode == MODE_CLASSIFIERS:
+            result += [self.rules_file_name, self.rules_directory,
+                       self.rules_class]
+            try:
```

Comment:
This shouldn't be necessary - the choices should be automatically populated if you make the changes to `get_rules_class_choices`